Detectors/TPC/workflow/CMakeLists.txt

```diff
@@ -200,6 +200,11 @@ o2_add_executable(calib-gainmap-tracks
                   SOURCES src/tpc-calib-gainmap-tracks.cxx
                   PUBLIC_LINK_LIBRARIES O2::TPCWorkflow O2::GlobalTracking)
 
+o2_add_executable(integrate-clusters
+                  COMPONENT_NAME tpc
+                  SOURCES src/tpc-integrate-cluster-currents
```

Comment:
Sorry to necrobump this PR, just saw that the file ending was missed here.
Now CMake produces the following warning:
![image](https://user-images.githubusercontent.com/42321018/205666924-335be88e-92bf-4de5-82f4-56c1204ec729.png)

Reply:
Thank you for noticing, I will fix it.